c++/src/capnp/compat/json.c++

```diff
@@ -379,13 +379,119 @@ void JsonCodec::encodeField(StructSchema::Field field, DynamicValue::Reader inpu
   encode(input, field.getType(), output);
 }
 
+namespace {
+int64_t parseInt64(kj::StringPtr s)
```

Comment:
Kenton, would you accept to merge these parse functions somewhere into `kj`? There is no json dependency in this code. It just  `kj` style wrapper around std function.